api/interactives.go

```diff
@@ -172,38 +164,39 @@ func (api *API) UpdateInteractiveHandler(w http.ResponseWriter, req *http.Reques
 		return
 	}
 
-	// 2.1 - fail if attempting to update the slug for a published model
-	if *existing.Published && update.Interactive.Metadata != nil && existing.Metadata.HumanReadableSlug != update.Interactive.Metadata.HumanReadableSlug {
+	// 4. fail if attempting to update the slug for a published model
+	if *existing.Published && formDataRequest.Update.Interactive.Metadata != nil && existing.Metadata.HumanReadableSlug != formDataRequest.Update.Interactive.Metadata.HumanReadableSlug {
 		http.Error(w, ErrCantUpdateSlug.Error(), http.StatusForbidden)
-		log.Error(ctx, fmt.Sprintf("attempting to update slug for a published model existing (%s), update (%s)", existing.Metadata.HumanReadableSlug, update.Interactive.Metadata.HumanReadableSlug), ErrCantUpdateSlug)
+		logMsg := fmt.Sprintf("attempting to update slug for a published model existing (%s), update (%s)", existing.Metadata.HumanReadableSlug, formDataRequest.Update.Interactive.Metadata.HumanReadableSlug)
+		log.Error(ctx, logMsg, ErrCantUpdateSlug)
 		return
 	}
 
-	// 3. prepare updated model
+	// 5. prepare updated model
 	updatedModel := &models.Interactive{
 		ID:            id,
-		Published:     update.Interactive.Published,
+		Published:     formDataRequest.Update.Interactive.Published,
```

Comment:
cosmetic - maybe assign formDataRequest.Update to a variable and use it (will save you using the long form)